data/migrations/20191907012229_action-table.js

```diff
@@ -0,0 +1,31 @@
+exports.up = function(knex, Promise) {
+    return knex.schema.createTable('actions', function(tbl) {
+        
+      tbl.increments()
+      .unique()
+  
+      tbl
+        .string('description')
+        .notNullable()
+
+      tbl
+        .string('notes')   
+        
+      tbl
+        .integer('project_id') 
```

Comment:
Nice work! So the relationship exists in the table but you're not optimizing your endpoint to POST an action properly. Instead of adding the 'project_id` manually through insomnia, it will be much cleaner if you already include it in your endpoint. You can do this by re-factoring your POST an action router to use the URL `/api/projects/:id/actions` where you can take the parameter id (that belongs to the desired project) and include it in your request body like this:

```
const project_id = req.params.id;
const newActionObject = req.body;
const action = await db('actions').insert({ ...newActionObject, project_id  });
res.status(201).json(action)
```